chaincode/permissions.go

```diff
@@ -26,14 +27,15 @@ const (
 	Process  Action = "process"
 	Download Action = "download"
 )
+*/
 
 // Permission represents one permission based on an action type
 type Permission struct {
 	// Public is true if this permission is given to the asset's owner only and
 	// the nodes listed in AuthorizedIDs (open to all nodes if false)
 	Public bool `json:"public"`
 	// AuthorizedIDs list all authorised nodes other than the asset's owner
-	AuthorizedIDs []string `json:"authorizedIDs", omitempty`
+	AuthorizedIDs []string `json:"authorizedIDs,omitempty"`
```

Comment:
This change is a nice catch but it belongs in its own PR and requires to update the tests since it breaks some of them.